packages/material/scss/tooltip/_theme.scss

```diff
@@ -0,0 +1,11 @@
+$tooltip-color: $base-bg !default;
```

Comment:
According to https://material.angular.io/components/tooltip/examples and https://material.io/guidelines/components/tooltips.html, tooltips are always white text on dark background (gray-700).